src/main/webapp/js/user-page-loader.js

```diff
@@ -92,6 +92,7 @@ function buildMessageDiv(message) {
 
 /** Fetches data and populates the UI of the page. */
 function buildUI() {
+  ClassicEditor.create( document.getElementById('message-input') );
```

Comment:
Code seems fine syntactically, but it is unclear what the new code you added actually does compared to what was already there.

Reply:
Sorry for the lack of description. This line of javascript comes from the CKEditor. It's what changes the text editor from a plain text editor to a rich text editor. When running the website you should now be able to do basic html editing when writing something in the text box.